### PR TITLE
Contract setup `Completed` is the same for taker and maker

### PR DESCRIPTION
**Pull request by da-kami**
Either of the roles has to record the outcome of the setup, which should be the same for them.
In order to handle this in our event model we need `Completed` in the model so it is not role specific.

### CODE REVIEW

**Review by thomaseizinger — APPROVED** [review]
ACK

Wait for @luckysori's approval before merge pls.

**Review by luckysori — APPROVED** [review]
LGTM!

**Comment by thomaseizinger** [comment]
@klochowicz Seems there is still something not quite right with the timeouts :(

**Comment by klochowicz** [comment]
bors retry

**Pull request merged**
